tests/unit/dataactvalidator/test_d43_detached_award_financial_assistance_1.py

```diff
@@ -0,0 +1,39 @@
+from tests.unit.dataactcore.factories.staging import DetachedAwardFinancialAssistanceFactory
+from tests.unit.dataactvalidator.utils import number_of_errors, query_columns
+
+_FILE = 'd43_detached_award_financial_assistance_1'
+
+
+def test_column_headers(database):
+    expected_subset = {"row_number", "place_of_performance_code", "place_of_perform_country_c"}
+    actual = set(query_columns(_FILE, database))
+    assert expected_subset == actual
+
+
+def test_success(database):
+    """ If PrimaryPlaceOfPerformanceCode is not USA, Congressional District must be blank """
+
+    det_award_1 = DetachedAwardFinancialAssistanceFactory(place_of_perform_country_c="Nk",
+                                                          place_of_performance_congr="")
+    det_award_2 = DetachedAwardFinancialAssistanceFactory(place_of_perform_country_c="CA",
+                                                          place_of_performance_congr="")
+    det_award_3 = DetachedAwardFinancialAssistanceFactory(place_of_perform_country_c="Mx",
+                                                          place_of_performance_congr="")
+    det_award_4 = DetachedAwardFinancialAssistanceFactory(place_of_perform_country_c="USA",
```

Comment:
Check this with a different capitalization of this as well. `usa` or `UsA` or something, just not all caps